index.html

```diff
@@ -0,0 +1,656 @@
+          </div>
+          <div class="nav-right">
+            <button type="button">SUBSCRIBE NOW</button>
+            <button type="button">LOG IN</button>
+          </div>
+        </nav>
+
+        <div class="main-content">
```

Comment:
This one is optional but worth doing. Let's use the `<main>` HTML tag instead of the div with the class of main here. This will make your code [more readable, accessible and semantic](https://medium.com/@zac_heisey/7-alternatives-to-the-div-html-tag-7c888c7b5036)! :muscle: